taxonomies/ingredients.txt

```diff
@@ -7978,6 +7978,10 @@ de:Milch aus Frankreich
 fr:lait origine france
 nl:melk uit Frankrijk
 
+<en:milk
+en:origin of the milk
+es:origen de la leche
+
```

Comment:
This doesn't seem to belong to the ingredients taxonomy.

Reply:
This is a temporary fix to fix #3570

Reply:
We shouldn't do that. We don't want to hide issues with the parsing, or to take temporary shortcuts. Taking shortcuts makes it harder to get the right solution in place.

That's also the reason why we don't add mispellings in the ingredients taxonomy: it would reduce the number of unrecognized ingredients, but it also makes it harder to have correct ingredients lists.